app/models/user.rb

```diff
@@ -393,6 +393,19 @@ def onboarding
     @onboarding ||= VeteranOnboarding.for_user(self)
   end
 
+  def demographics_info
+    # return nil unless DemographicsPolicy.new(self).access? && MPIPolicy.new(self).queryable?
```

Comment:
This is where policy checks could be made. Would potentially satisfy department-of-veterans-affairs/va.gov-team/issues/79924